### DISCUSSION TITLE
Moving the blog to Elixir and Phoenix LiveView - Post

### DISCUSSION THREAD

**Discussion by dbernheisel** [dbernheisel/bernheisel.com#4] [General]
![moving-blog](https://user-images.githubusercontent.com/643967/103720141-ae21b500-4f98-11eb-8963-610172f4f6ec.png)

https://bernheisel.com/blog/moving-blog

Have reactions to the post? Let's discuss!

**Comment by georgethomas07** [comment]
Thank you for taking the time to write this informative article. I just started getting my blog out there and the advice you offer sounds very helpful, so I will definitely be following your future posts! [https://colaninfotech.com/](url)

**Reply by dbernheisel** [reply]
thank you!
